app/src/main/java/Activities/BrowseUserProblems.java

```diff
@@ -77,6 +77,7 @@ public void onItemClick(AdapterView<?> l, View v, int position, long id){
         // Get position of clicked item and pass it on to Item Activity for later processing
         Intent intent = new Intent(BrowseUserProblems.this,ViewProblemActivity.class);
         intent.putExtra("position", position);
+        intent.putExtra("USERIDEXTRA",this.userId);
```

Comment:
remove quotations on USERIDEXTRA to maintain consistency

Reply:
USERIDEXTRA is imported from Global variables. If you check out the files Michael made that contains all global variables, it should be there